src/main/java/duck/storage/LocalStorage.java

```diff
@@ -0,0 +1,75 @@
+package duck.storage;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+import duck.exception.DuckException;
+import duck.task.TaskList;
+
+/**
+ * LocalStorage makes use of Serializable objects to write to a local file.
+ */
+public class LocalStorage implements Storage {

```

Comment:
Really neat exemplification of abstraction with interfaces for Storage implemented by main class LocalStorage!